File: doc/codes/python/examples/api/command/gpio.py
from autd3.commands import GpioOut, SetGpioOut

gpio0 = GpioOut.PatternBank
gpio1 = GpioOut.Thermo
gpio2 = GpioOut.PwmOut(0)
gpio3 = GpioOut.Off
outputs = [gpio0, gpio1, gpio2, gpio3]
# ANCHOR: api
SetGpioOut(outputs=outputs)
# ANCHOR_END: api

File: doc/codes/rust/examples/api/command/gpio.rs
use autd3_rs::commands::{GpioOut, SetGpioOut};

fn main() {
    let gpio0 = GpioOut::PatternBank;
    let gpio1 = GpioOut::Thermo;
    let gpio2 = GpioOut::PwmOut(0);
    let gpio3 = GpioOut::Off;
    let outputs = [gpio0, gpio1, gpio2, gpio3];
    // ANCHOR: api
    SetGpioOut { outputs };
    // ANCHOR_END: api
}
